doubly_linked_list/doubly_linked_list.py

```diff
@@ -44,16 +44,48 @@ def __len__(self):
     return self.length
 
   def add_to_head(self, value):
-    pass
+    if self.head is None:
+      new_node = ListNode(value)
+      self.head = new_node
+      return
+    new_node = ListNode(value)  
+    new_node.next = self.head
+    self.head.prev = new_node
+    self.head = new_node
 
   def remove_from_head(self):
-    pass
+    if self.head is None:
+      print("The List Has Nothing To Delete")
+      return 
+    if self.head.next is None:
+      self.head = None
+      return
+    self.head = self.head.next
+    self.start_prev = None
```

Comment:
Do these tests pass? Because I'm pretty sure you need to return the value of the node that you're removing. And that would mean you would need to hold that node in a variable.